bin/__tests__/carbon-codemod.js

```diff
@@ -202,5 +202,30 @@ describe("run", () => {
       expect(execaSync.mock.calls[1][0]).toEqual(Cli.__jsCodeShiftBin);
       expect(execaSync.mock.calls[1][1]).toEqual(args);
     });
+
+    it("runs message-remove-classic-theme", () => {
+      process.argv = [
+        "/Users/jamime/.nvm/versions/node/v10.16.3/bin/node",
```

Comment:
Just because I can't remember - why are these using your username again?

Reply:
It just needs to be a realistic path, I'll refactor this to be `guest`.